modules/auxiliary/server/browser_autopwn.rb

```diff
@@ -236,9 +236,9 @@ def setup
       print_debug("NOTE: Debug Mode; javascript will not be obfuscated")
     else
       pre = Time.now
-      print_status("Obfuscating initial javascript #{pre}")
-      @init_js.obfuscate
-      print_status "Done in #{Time.now - pre} seconds"
+      #print_status("Obfuscating initial javascript #{pre}")
```

Comment:
That would be a good idea. But I need you to try first before I conclude that's really the problem. So let me know, please.